### PR TITLE
updating template

### PR DESCRIPTION
**Pull request by Sgtpluck**
Updating eng-hiring to use https://github.com/18F/isildurs-bane template.

### CODE REVIEW

File: _data/footer.yml [hunk 1]
```diff
@@ -1,43 +1,2 @@
...
-big_footer_signup_form: false
-
-# Configuration for agency logo(s) (shown side by side).
-# If the logo is external add external: true
-logos:
-  - src: https://18f.gsa.gov/assets/img/logos/18f-logo.svg
-    alt: 18F
-    external: true
-
-# Configuration for footer contact links
```
[the remark this answers is not in the document]
Comment (lauraGgit) [reply _data/footer.yml:37]: discussed and will hold off for now

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: Daily Sync
```
[the remark this answers is not in the document]
Comment (lauraGgit) [reply .github/workflows/main.yml:1]: discussed and yes!

**Review by lauraGgit — APPROVED** [review]
🚀 LGTM! looks good and closes #91

File: _config.yml [hunk 3]
```diff
@@ -1,52 +1,37 @@
...
     permalink: /:path/
 
 defaults:
-  -
-    scope:
+  - scope:
       path: ""
     values:
-      layout: page
+      layout: post
```
[the remark this answers is not in the document]
Comment (Sgtpluck) [reply _config.yml:28]: it is part of the guide template!

**Comment by adunkman** [comment]
Puppy running through the grass and tackling the camera joyously

This is so great! This was something that was on my list from September and hadn’t found the time to get it done — and then handed it off. 

🎉 👏  from the sidelines!

**Pull request merged**